### ISSUE TITLE
MT7986: How to do pre-Calibration?

### ISSUE THREAD

**Issue by Petertw01** [openwrt/mt76#918]
Hi~

I need some help. Since Mac80211 does not use **iwpriv** so it cannot use ATE commands to do the calibration. 
So how to do the pre-calibration? Do the mac80211 drivers and chipsets might support specific calibration commands?

Thanks

**Comment by nachalni** [comment]
> Do the mac80211 drivers and chipsets might support specific calibration commands?

No, it doesn't, but there is support for reading and applying it for some chips, which is very nice

> So how to do the pre-calibration? 

You have to use mtk's proprietary driver to do the precal
